tests/scsi/012

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2025 Google LLC
+
+. common/null_blk
+. common/scsi_debug
+. tests/scsi/rc
+
+DESCRIPTION="test scsi_debug timeout and abort handling"
+QUICK=1
+
+requires() {
+	_have_scsi_debug
```

Comment:
Nit: it would be the better to have "_have_program getconf" here to be consistent with zbd/014.

Reply:
Done.